src/Homework1/program6.py

```diff
@@ -0,0 +1,9 @@
+# 6 lines: Import, regular expressions
+
+
+import re
```

Comment:
here you have 0 lines between imports and code (earlier - 1). Your code should be consistent

Reply:
Left 1 line beetween import and code in all files